python/pyarrow/src/arrow/python/datetime.h

```diff
@@ -20,9 +20,13 @@
 #include <algorithm>
 #include <chrono>
 
+#include "arrow/python/platform.h"
+#include "arrow/python/visibility.h"
```

Comment:
I suppose this is a left-over from rebasing on top of the moved code (those lines can be removed, they are already included a bit below as well)